client/service-worker.js

```diff
@@ -1,3 +1,5 @@
+importScripts('javascript/sw.js');
```

Comment:
Let's use ECMA import instead of this.
For example,
`import Push from './push/push_manager.js';`